Misc/NEWS.d/next/C API/2022-05-03-19-35-37.gh-issue-92193.61VoFL.rst

```diff
@@ -0,0 +1,6 @@
+CPython extensions providing optimized execution of Python bytecode (like Cinder JIT and Pyjion)
+can benefit from being able to modify the vectorcall field on instances of PyFunctionObject to allow calling the optimized path (e.g. JIT-compiled) directly.
+
+This PR introduces an API call where vectorcall field can be modified like so:

```

Comment:
This text lands into https://docs.python.org/dev/whatsnew/changelog.html#changelog which is a changlog, you should not mention "a PR" there.

I suggest to just copy/paste what you wrote in whatsnew.